src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        I am
+        {' '}
+        {person.age}
+      </p>
+    )}
+
+    {person.isMarried
+      ? (
+        <p className="Person__partner">
+          {person.partnerName}
+          {' '}
+          is my
+          {' '}
+          {person.sex === 'm' ? 'wife' : 'husband'}
+        </p>
+      )
+      : (
+        <p className="Person__partner">
+          I am not married
+        </p>
```

Comment:
Move a ternary operator inside the `<p className="Person__partner">` tag

Reply:
done